dev/ledger.md

```diff
@@ -583,6 +621,49 @@ and contains the following fields:
   in the reference implementation, one of these two fields will be zero
   in that case.
 
+- If this is an `ApplicationCall` transaction, the `EvalDelta` associated with the successful execution of the corresponding application's `ApprovalProgram` or `ClearStateProgram`. The `EvalDelta`, encoded as msgpack field `dt`, contains the following fields:
+  - A `GlobalDelta`, encoding changes to the global state of the called application, encoded as msgpack field `gd`.
+    - `gd` is a [`StateDelta`][State Deltas].
+  - Zero or more `LocalDeltas`, encoding changes to some local states associated with the called applicaiton, encoded as msgpack field `ld`.
+    - `ld` maps an "account offset" to a [`StateDelta`][State Deltas]. Account offset 0 is the transaction's sender. Account offsets 1 and greater refer to the account specified at that index minus one in the transaction's `Accounts` slice.
+
+TEAL Key/Value Stores
+---------------------
```

Comment:
It might make sense to have this as a subsection of `Applications`.

Reply:
done